src/test/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravExpiredVarselSpek.kt

```diff
@@ -73,6 +75,35 @@ class AktivitetskravExpiredVarselSpek : Spek({
                 personOppgave.type shouldBeEqualTo PersonOppgaveType.AKTIVITETSKRAV_VURDER_STANS
                 personOppgave.personIdent shouldBeEqualTo ARBEIDSTAKER_FNR
             }
+            it("Consumes expired varsel creates personoppgave but not duplicate") {
```

Comment:
Ble ikke helt ferdig i dag, så skal legge til en test til i neste PR (for at det lages en oppgave hvis det finnes en fra før som er behandlet).